.coveragerc

```diff
@@ -0,0 +1,3 @@
+[run]
+omit=tests/*, .venv/*
```

Comment:
Should this specify `gatorgrouper/tests/*`? Also, is this for codecov or the old coverage? It may need to be deleted.

Reply:
I believe it is for `codecov` as omitting `.venv/*` worked for me previously. However I will look into its need and correct path if necessary.